velox/core/QueryConfig.h

```diff
@@ -355,7 +355,7 @@ class QueryConfig {
   // Returns the target size for a Task's buffered output. The
   // producer Drivers are blocked when the buffered size exceeds
   // this. The Drivers are resumed when the buffered size goes below
-  // PartitionedOutputBufferManager::kContinuePct % of this.
+  // OutputBufferManager::kContinuePct % of this.
   uint64_t maxPartitionedOutputBufferSize() const {
```

Comment:
Do we need to rename this property as well?

maxPartitionedOutputBufferSize -> maxOutputBufferSize